data/crac/crac-impl/src/main/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImpl.java

```diff
@@ -108,9 +109,16 @@ public double getMaxAdmissibleSetpoint(double previousInstantSetPoint) {
 
     @Override
     public void apply(Network network, double targetAngle) {
-        PhaseTapChanger phaseTapChanger = getPhaseTapChanger(network);
+        getPhaseTapChanger(network);
```

Comment:
Do you think it is necessary to keep this line, or can we rely on powybl to throw an exception it the pst doesn't exist? (knowing that, in most cases, the Crac is filtered by cross checking it with the network, during import)

Reply:
Yes powsybl can raise exception but I need to change the apply I used: .apply(network) -> .apply(network, true, ReportNode.NO_OP)
I think I need to use this apply for other range actions too, is that correct?

Reply:
yes it is safer

Reply:
done